services/outputhost/extcache.go

```diff
@@ -431,3 +522,61 @@ func (extCache *extentCache) getState() *admin.OutputCgExtent {
 
 	return cge
 }
+
+// KafkaGroupMetadata is a structure used for JSON encoding/decoding of the metadata stored for
+// Kafka groups joined by Cherami
+type KafkaGroupMetadata struct {
+	// Version is the version of this structure
+	Version uint
+
+	// CGUUID is the internal Cherami consumer group UUID that committed this offset
+	CGUUID string
+
+	// OutputHostUUID is the UUID of the Cherami Outputhost that committed this offset
+	OutputHostUUID string
+}
+
+const kafkaGroupMetadataVersion = uint(0) // Current version of the KafkaGroupMetadata
+
+func kafkaNotificationsLogger(ch <-chan *sc.Notification, log bark.Logger) {
+	notificationNum := 0
+notificationsLoop:
+	for {
+		select {
+		case n, ok := <-ch:
+			if !ok {
+				break notificationsLoop
+			}
+			if n == nil {
+				log.Warn(`nil notification received`)
+				continue notificationsLoop
+			}
+			notificationNum++
+			log = log.WithField(`notificationNum`, notificationNum)
+			if len(n.Claimed) > 0 {
+				log.WithField(`claimed`, n.Claimed).Info(`claimed partitions`)
+			}
+			if len(n.Released) > 0 {
+				log.WithField(`released`, n.Released).Info(`released partitions`)
+			}
+			if len(n.Current) > 0 {
+				log.WithField(`current`, n.Released).Info(`current partitions`)
+			}
+		}
+	}
+	log.Info(`Notifications channel closed`)
+}
+
+func getKafkaGroupIDForCheramiConsumerGroupName(cgName string) string {
+	s := strings.Split(cgName, `/`)
```

Comment:
Followed up with the Kafka team; they discourage using '/' in consumer group names, even if it will work.